x/auth/vesting/keeper/keeper.go

```diff
@@ -0,0 +1,55 @@
+package keeper
+
+import (
+	storetypes "github.com/cosmos/cosmos-sdk/store/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/auth/vesting/exported"
+	"github.com/cosmos/cosmos-sdk/x/auth/vesting/types"
+)
+
+type VestingKeeper struct {
+	accountKeeper types.AccountKeeper
+	bankKeeper    types.BankKeeper
+	key           storetypes.StoreKey
+}
+
+func NewVestingKeeper(
+	accountKeeper types.AccountKeeper,
+	bankKeeper types.BankKeeper,
+	key storetypes.StoreKey,
+) VestingKeeper {
+	return VestingKeeper{
+		accountKeeper,
+		bankKeeper,
+		key,
+	}
+}
+
+// AddVestingAccount adds the address of vesting account to store.
+// The caller should check the account type to make sure it's a vesting account type.
+func (vk VestingKeeper) AddVestingAccount(ctx sdk.Context, addr sdk.AccAddress) {
+	store := ctx.KVStore(vk.key)
+	store.Set(types.VestingAccountStoreKey(addr), []byte{})
```

Comment:
Why is this getting set to empty bytes here? Why not set the actual value?

Reply:
this is simply used for the convenience of iterating vesting accounts